server.js

```diff
@@ -1,38 +1,34 @@
 // TASK 1: Setting up server
 
 
-
 const http = require('http');
 
-// Creating the HTTP server
 const server = http.createServer((req, res) => {
-
```

Comment:
The MongoDB URI should be stored as an environment variable rather than hardcoding it in the code. This is not secure and can potentially expose sensitive information.